nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
```

Comment:
- [ ] Kindly edit the exception to make sure the error raised has a message displayed which is a good practice when using the method 👍 

### Sample:
```
 class Nameable
    def correct_name
        raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
    end
end

```